docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,8 +791,531 @@ To read and write a value of the Bluetooth descriptor:
+     serviceUuid: '5678',
+     isPrimary: true
+   };
+
+   var serviceInitData = {
+     serviceUuid: '1234',
+     includedServices: [includedServiceInitData],
+     characteristics: [characteristicInitData],
+     isPrimary: true
+   };
+   ```
+
+2. Register the service in the local server:
+
+   ```
+   function successCallback() {
+     console.log('Service registered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Registering GATT service failed: ' + error.message);
+   }
+
+   var server = tizen.bluetooth.getGATTServer();
+   server.registerService(serviceInitData, successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > The service will be unregistered from the server when:
+   > - service's `unregister()` method is called,
+   > - server's `unregisterAllServices()` method is called,
+   > - application is reloaded,
+   > - application is closed.
+
+### Unregister services
+GATT services can be unregistered either one at a time or all at once.
+
+To unregister a single service from the local GATT server:
+
+1. Choose the service to be unregistered:
+   ```
+   var server = tizen.bluetooth.getGATTServer();
+   var serviceToBeUnregistered = server.services[3]; 
+   ```
+
+2. Call `unregister()` method:
+   ```
+   function successCallback() {
+     console.log('Service unregistered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Unregistering GATT service failed: ' + error.message);
+   }
+
+   serviceToBeUnregistered.unregister(successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > After `unregister()` is called for the last registered service, the server is stopped.
+   > If new services are then registered, server's `start()` method has to be
+   > called to make them visible to clients.
+
+To unregister all services from the local GATT server at once, call server's
+`unregisterAllServices()` method:
+
+   ```
+   function successCallback() {
+     console.log('All services unregistered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Unregistering all GATT services failed: ' + error.message);
+   }
+
+   var server = tizen.bluetooth.getGATTServer();
+   server.unregisterAllServices(successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > After the calling `unregisterAllServices()`, the server is stopped.
+   > If new services are then registered, server's `start()` method has to be
+   > called to make them visible to clients.
+
+### Send notifications about characteristic's value changes to the clients
+GATT clients connected to the server running on the local device can register for updates of its characteristics' values.
+The server can send two types of such updates - _notifications_ and _indications_.
+They differ in that, clients receiving indications have to acknowledge them - i.e. send a message back to the server, telling that they received the new value. Clients do not acknowledge notifications.
```

Comment:
@surbhig12 Would you please check this sentence again?

They differ in that, clients receiving indications have to acknowledge them - i.e. send a message back to the server, telling that they received the new value. Clients do not acknowledge notifications.

Reply:
Please let me know if the followign sentence is technically fine:

The difference in notifications and indications is that clients receiving indications have to acknowledge them. This means that the client must send a message back to the server, telling that they received the new value. Clients do not acknowledge notifications.

Please verify if the last sentence is correct: Clients do not acknowledge notifications.

Please verify for technical accuracy.

Reply:
@surbhig12, The sentences you have proposed are technically correct.

I have one question about the second sentence:
> This means that the client must send a message back to the server, telling that they received the new value.

Shouldn't clients be plural here, like:
> This means that the _clients_ must send a message back to the server, telling that they received the new value.
?

For now, I change the sentence to the version with plural clients. If you have meant singular `the client`, please let me know and I'll fix it.

Reply:
Thank you for the clarification, the rephrased sentence is fine.